tools-v2/pkg/cli/command/curvebs/snapshot/copyset/copyset.go

```diff
@@ -56,97 +60,202 @@ func (sRpc *SnapshotRpc) Stub_Func(ctx context.Context) (interface{}, error) {
 	return sRpc.Client.Snapshot(ctx, sRpc.Request)
 }
 
-type SnapshotOneCommand struct {
+type SnapshotAllRpc struct {
+	Info    *basecmd.Rpc
+	Request *cli2.SnapshotAllRequest
+	Client  cli2.CliService2Client
+}
+
+func (sRpc *SnapshotAllRpc) NewRpcClient(cc grpc.ClientConnInterface) {
+	sRpc.Client = cli2.NewCliService2Client(cc)
+}
+
+func (sRpc *SnapshotAllRpc) Stub_Func(ctx context.Context) (interface{}, error) {
+	return sRpc.Client.SnapshotAll(ctx, sRpc.Request)
+}
+
+type SnapshotCopysetCommand struct {
 	basecmd.FinalCurveCmd
 
-	Rpc      *SnapshotRpc
-	Response *cli2.SnapshotResponse2
-	row      map[string]string
+	Rpc     *SnapshotRpc
+	AllRpcs []*SnapshotAllRpc
+	needAll bool
+	rows    []map[string]string
+
+	timeout    time.Duration
+	retryTimes int32
 }
 
-var _ basecmd.FinalCurveCmdFunc = (*SnapshotOneCommand)(nil) // check interface
+var _ basecmd.FinalCurveCmdFunc = (*SnapshotCopysetCommand)(nil) // check interface
 
 // NewCommand ...
-func NewSnapshotOneCommand() *cobra.Command {
-	peerCmd := &SnapshotOneCommand{
+func NewCopysetCommand() *cobra.Command {
+	return NewSnapshotCopysetCommand().Cmd
+}
+
+func NewSnapshotCopysetCommand() *SnapshotCopysetCommand {
```

Comment:
it seems that we don't need `NewCopysetCommand`, u can change snapshot.go to directly use `NewSnapshotCopysetCommand`.

Reply:
done